### PR TITLE
Handles extra parameter sanitizeDisable for saving block(s)

### PR DESCRIPTION
**Pull request by sos-productions**
Fullfill my own request described here:https://github.com/codex-team/editor.js/issues/1280

### CODE REVIEW

File: build [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+#yarn has to be installed first
+#
+npm run-script build
```
Comment (hata6502) [inline build:4]: There are some unrelated files.

File: src/components/modules/api/saver.ts [hunk 2]
```diff
@@ -14,14 +14,16 @@ export default class SaverAPI extends Module {
    */
   public get methods(): Saver {
     return {
-      save: (): Promise<OutputData> => this.save(),
+      save: (sanitizeDisable = true): Promise<OutputData> => this.save(sanitizeDisable),
```
Comment (hata6502) [inline src/components/modules/api/saver.ts:17]: I think default sanitizeDisable should be `false` for safety.

**Review by hata6502 — COMMENTED** [review]
I think the undo function is important for Editor.js :+1:

File: build (hunk 1 above)
Comment (sos-productions) [reply build:4]: Helper file to build optional but I submitted an issue about github forking not complete where I completed this script to be able to build editorjs. see https://github.com/codex-team/editor.js/issues/1283

File: src/components/modules/api/saver.ts (hunk 2 above)
Comment (sos-productions) [reply src/components/modules/api/saver.ts:17]: yes you are right

File: build (hunk 1 above)
Comment (hata6502) [reply build:9]: I read https://github.com/codex-team/editor.js/issues/1283, and commented. 
Editor.js uses git sub modules. 
https://github.com/codex-team/editor.js/issues/1283#issuecomment-680714370

File: src/components/modules/api/saver.ts (hunk 2 above)
Comment (sos-productions) [reply src/components/modules/api/saver.ts]: @hata6502 As I end up discussion https://github.com/codex-team/editor.js/issues/1280 sanitizing should better be focus on input and on server side, it is good however that a first stage of sanitizing on pasting, rendering. and on inserting blocks has been done but for large number of blocks this overload on client side gives this https://github.com/codex-team/editor.js/issues/1191 . However, this patch that imho has no security issue because we work not input but on output, readonly for block content .

**Comment by neSpecc** [comment]
Outdated
